src/Uno.UWP/Devices/Enumeration/Internal/Providers/ProximitySensor/ProximitySensorDeviceClassProvider.Android.cs

```diff
@@ -0,0 +1,61 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using Android.Hardware;
+using Uno.Devices.Enumeration.Internal;
+using Uno.Devices.Sensors.Helpers;
+
+namespace Windows.Devices.Enumeration.Internal.Providers.ProximitySensor;
+
+internal class ProximitySensorDeviceClassProvider : IDeviceClassProvider
+{
+	public bool CanWatch => false;
+
+#pragma warning disable CS0067
+	public event EventHandler<DeviceInformation>? WatchAdded;
+
+	public event EventHandler<DeviceInformation>? WatchEnumerationCompleted;
+
+	public event EventHandler<DeviceInformationUpdate>? WatchRemoved;
+
+	public event EventHandler<object>? WatchStopped;
+
+	public event EventHandler<DeviceInformationUpdate>? WatchUpdated;
+#pragma warning restore CS0067
+
+	public Task<DeviceInformation[]> FindAllAsync() => Task.Run(() =>
```

Comment:
Why do we need Task.Run?

Reply:
It was suggested by Jerome above, it is not strictly necessary, but won't hurt, as the method is `Task` based anyway